app/assets/javascripts/Components/Modals/send_lti_grades_modal.jsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import Modal from "react-modal";
+
+class LtiGradeModal extends React.Component {
+  static defaultProps = {
+    override: false,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      deployments: this.props.lti_deployments,
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentDidMount() {
+    Modal.setAppElement("body");
+    const deploymentsMapped = this.state.deployments.map(deployment => ({
+      deployment: deployment.id,
+      checked: true,
+    }));
+    this.setState({deploymentsChecked: deploymentsMapped});
```

Comment:
This should have a default value set in the constructor. Even if you want it to be explicitly undefined, write that in the constructor so that the component is aware of the potential state keys from the beginning.